wandb/apis/public/artifacts.py

```diff
@@ -318,12 +319,18 @@ def __init__(
         self.client = client
         self.entity = entity
         self.project = project
-        self.name = name
-        self.type = type
+        self._name = name
+        self._saved_name = name
+        self._type = type
+        self._saved_type = type
         self._attrs = attrs
         if self._attrs is None:
             self.load()
         self._aliases = [a["node"]["alias"] for a in self._attrs["aliases"]["edges"]]
+        self._description = self._attrs["description"]
+        self._tags = [a["node"]["name"] for a in self._attrs["tags"]["edges"]]
+        self._saved_tags = copy(self._tags)
+        self._is_sequence = self.load_is_sequence()
```

Comment:
Good idea to load sequence on init, but let's wrap it into `self.load()` and extend that query instead of adding a new function for it; it's a very small addition to that query, and that way we're making 1 round trip instead of 2.